ui/src/App.tsx

```diff
@@ -29,6 +29,9 @@ function App(): JSX.Element {
             <Route path="/login">
               <Login />
             </Route>
+            <Route path="/upload">
```

Comment:
Should this be nested under admin? `admin/upload`?

Reply:
Resolving this, will nest it after Login gets merged.